content/library/api/connections/connections.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Connections and databases
+slug: /library/api-reference/connections
+---
+
+# Connections and databases
+
+## Setup your connection
+
+<TileContainer>
+<RefCard href="/library/api-reference/connections/st.experimental_connection" size="half">
+
+<Image pure alt="screenshot" src="/images/databases/experimental-connection.png" />
+
+#### Create a connection
+
+Connect to a database and return a connection object.
```

Comment:
`Connect to a data source or API`

Reply:
Fixed on 16e925e

Reply:
Sorry it wasn't clear - I meant to change the description, not the heading. I think it should be:

```markdown
#### Create a connection

Connect to a data source or API.
```

Reply:
Good catch, my bad! Fixed on b944faf